packages/preferences-controller/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- **BREAKING:** Rename openSeaEnabled to displayNftMedia ([#4774](https://github.com/MetaMask/core/pull/4774))
+- **BREAKING:** Rename setOpenSeaEnabled to setDisplayNftMedia ([#4774](https://github.com/MetaMask/core/pull/4774))
```

Comment:
Even though they are marked as BREAKING, these changes are already patch in the mobile client, so updating the controller and removing the changes from the mobile patch will not require any additional changes.